src/cirrus/lib/utils.py

```diff
@@ -296,6 +296,90 @@ def delete_from_queue_batch(messages: list[dict]) -> dict:
     return resp
 
 
+def build_item_sns_attributes(item: dict) -> dict:  # noqa: C901
```

Comment:
This was taken from the old publish builtin task:
https://github.com/cirrus-geo/cirrus-geo/blob/v0.15.4/src/cirrus/builtins/tasks/publish/lambda_function.py#L29

I made this a util function rather than a `ProcessPayload` method to a) avoid namespace pollution and b) because it operates on an `item` dict, not a payload.